zipline/data/data_portal.py

```diff
@@ -262,15 +251,17 @@ def _get_ctable(self, asset):
             else:
                 path = "{0}/{1}.bcolz".format(self._minutes_futures_path, sid)
         elif isinstance(asset, Equity):
-            if self._equity_sid_path_func is not None:
-                path = self._equity_sid_path_func(
-                    self._minutes_equities_path, sid
+            if self._equity_minute_reader.sid_path_func is not None:
+                path = self._equity_minute_reader.sid_path_func(
+                    self._equity_minute_reader.rootdir, sid
                 )
             else:
-                path = "{0}/{1}.bcolz".format(self._minutes_equities_path, sid)
+                path = "{0}/{1}.bcolz".format(
+                    self._equity_minute_reader.rootdir, sid)
 
         else:
-            path = "{0}/{1}.bcolz".format(self._minutes_equities_path, sid)
+            path = "{0}/{1}.bcolz".format(
```

Comment:
Not silly at all. It is the intended next step. This is just minimum scope
for testing the passing of the new parameter.

On Fri, Dec 4, 2015, 8:57 AM Richard Frank <notifications@github.com> wrote:

> In zipline/data/data_portal.py
> <https://github.com/quantopian/zipline/pull/896#discussion_r46683619>:
>
> >
> >          else:
> > -            path = "{0}/{1}.bcolz".format(self._minutes_equities_path, sid)
> > +            path = "{0}/{1}.bcolz".format(
>
> This may be a silly question, but if the data portal is now passed a bcolz
> reader, why does it also need to understand bcolz and where to read from? I
> would think that's abstracted away by the reader.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/quantopian/zipline/pull/896/files#r46683619>.
>